maquetado/application.css

```diff
@@ -0,0 +1,4 @@
+.title {
```

Comment:
Add css amd map files to gitignore, since they are being transpiled by sass, and keeping the original scss files is enough

Reply:
I agree.